app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+    
+    
+    rescue_from CanCan::AccessDenied do |exception|
+        redirect_to root_path
+    end    
 end
```

Comment:
Check the indentation of your file.

Reply:
Thanks i will be more careful with the indentation and spaces issues. I already made this change.